### PR TITLE
New Yeoman Cheat Sheet Goodies

### PR DESCRIPTION
**Pull request by StevenPyle**
## Description of new Instant Answer, or changes

This Instant Answer provides a cheat sheet to some basic commands to use with the Yeoman application framework.

## People to notify

---

Instant Answer Page: https://duck.co/ia/view/yeoman_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@StevenPyle Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/yeoman.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    "aliases": [
+    ],
+
+    "template_type": "terminal",
+
+    "section_order": [
+        "Getting started",
+        "Basic scaffolding",
+        "Other yo commands",
+        "Troubleshooting"
```
Comment (StevenPyle) [inline share/goodie/cheat_sheets/json/yeoman.json]: Sounds good.  I'll update with your recommendations, thank you for the feedback!

**Comment by StevenPyle** [comment]
Hi,

I merged in the changes recommended by jophab.  Is there anything else I need to do to get it back in the review queue?

Thanks!
Steve

**Comment by jophab** [comment]
@moollaza @tagawa Why beta giving this?
del

**Comment by StevenPyle** [comment]
@jobhap @moollaza @tagawa  I see that as well.  Searching on "yeoman cheat sheet" gives the github IA while "yeoman commands" returns this IA, correctly.

**Comment by moollaza** [comment]
@jophab the ID's for the IA page in this PR and that cheat sheet don't match. That's causing the issue.

The IA page for the yeoman cheat sheet should have the id `yeoman_cheat_sheet` instead of `yeoman_basic_commands`

**Comment by StevenPyle** [comment]
@moollaza I don't see a way to update the id on the IA page to set it to `yeoman_cheat_sheet`.  Am I missing something?

**Comment by pjhampton** [comment]
Hey @StevenPyle!

I have updated the title and id for you here: https://duck.co/ia/view/yeoman_cheat_sheet

**Comment by StevenPyle** [comment]
Thanks @pjhampton!
-Steve

**Comment by daxtheduck** [comment]
@StevenPyle this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**